model/event/subscriptions.go

```diff
@@ -246,6 +246,30 @@ func RemoveSubscription(id string) error {
 	})
 }
 
+func IsSubscriptionAllowed(sub Subscription) (bool, string) {
+	for _, selector := range sub.Selectors {
+		if selector.Type == "object" {
+			if selector.Data == "build" || selector.Data == "version" || selector.Data == "task" {
+				if sub.Subscriber.Type == "jira-issue" || sub.Subscriber.Type == "evergreen-webhook" {
```

Comment:
are there constants for these types?

Reply:
Wow there are so many constants in triggers that should be in subscriptions.